client/fingerprint/host.go

```diff
@@ -38,11 +35,7 @@ func (f *HostFingerprint) Fingerprint(cfg *config.Config, node *structs.Node) (b
 	node.Attributes["kernel.version"] = ""
 
 	if runtime.GOOS != "windows" {
```

Comment:
Yeah, I doubt it will ever be interesting on Windows but the fewer runtime.GOOS checks we have the happier I am! :)

Who knows, maybe someday gopsutils will populate a meaningful value for Windows.